app/views/welcome/index.html.erb

```diff
@@ -33,11 +33,27 @@
           <label class="form-label" for="AB_select">
             A/B test being used
           </label>
-          <select class="form-control" name="which_test" id="AB_select">
+          <select class="form-control" name="which_test" id="AB_select" onchange="showForm()">
             <option value="search_match_length">Search Match Length</option>
+            <option value="none">None</option>
           </select>
         </div>
 
+        <div class="host-wrapper" id="host-wrapper">
```

Comment:
This div is never displayed for users with JS disabled. A better approach is to have it visible on page load and then hide with JS if the control value is not selected. This also ensures it is correct displayed on page reload.

Reply:
Sorted! :)